.github/workflows/macos-workflow.yml

```
@@ -115,3 +124,55 @@ jobs:
       - name: Run Tests
         working-directory: build
         run: make -j$(getconf _NPROCESSORS_ONLN) unittests
+
+      - name: Prepare Artifact Metadata
+        id: artifact-metadata
+        shell: bash
```

This is the default shell.
```suggestion
        shell: bash
```